### PR TITLE
LPD-24943 Use the old version label as fallback when obtaining the inputStream when copying the file.

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
To reproduce follow steps on https://liferay.atlassian.net/browse/LPD-24943

- **LPD-24943 Use the old version label as fallback when obtaining the inputStream when copying the file.**
- **LPD-24943 extract method**
- **LPD-24943 add test to check the update of an old fashion store data that does not have the storeUUID on the file**

### CODE REVIEW

File: modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/service/test/DLCheckInCheckOutTest.java [hunk 1]
```diff
@@ -572,6 +633,25 @@ protected FileEntry updateFileEntry(long fileEntryId, String fileName)
...
+				dlFileEntry.getName()
+			).versionLabel(
+				dlFileVersion.getVersion()
+			).build(),
+			new UnsyncByteArrayInputStream(_TEST_CONTENT.getBytes()));
+		DLStoreUtil.deleteFile(
+			dlFileEntry.getCompanyId(), dlFileEntry.getDataRepositoryId(),
+			dlFileEntry.getName(), dlFileVersion.getStoreFileName());
+	}
+
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/service/test/DLCheckInCheckOutTest.java:654]: Can you rename this to something like `_renameDLStoreFile`.

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Review by adolfopa — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1407401_9454

**Comment by AliciaGarciaGarcia** [comment]
ci:report:2566570298

**Comment by AliciaGarciaGarcia** [comment]
ci:forward:force

**Pull request closed without merging**